bookkeeper-server/src/test/java/org/apache/bookkeeper/test/BookKeeperClusterTestCase.java

```diff
@@ -241,7 +241,12 @@ protected void stopAllBookies() throws Exception {
         for (BookieServer server : bs) {
             server.shutdown();
         }
+        bsConfs.clear();
         bs.clear();
+        if (bkc != null) {
```

Comment:
this seems unrelated to the change, does this impact the compaction tests ?
I am fine with this change anyway

Reply:
Yeah, this is changed for compaction tests.